bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/Runtime.java

```diff
@@ -45,7 +45,7 @@
 
     private Scheduler scheduler;
 
-    Runtime(Scheduler scheduler) {
+    public Runtime(Scheduler scheduler) {
```

Comment:
We cannot expose the Runtime constructor. Let's keep it private and duplicate code

Reply:
Made the constructor package-private in 0c17327